lib/vtls/cyassl.c

```diff
@@ -336,10 +342,12 @@ cyassl_connect_step1(struct connectdata *conn,
      https://github.com/wolfSSL/wolfssl/issues/366
      The supported curves below are those also supported by OpenSSL 1.0.2 and
      in the same order. */
+#ifndef WOLFSSL_TLS13
   CyaSSL_CTX_UseSupportedCurve(conssl->ctx, 0x17); /* secp256r1 */
```

Comment:
Most likely my `conssl` -> `BACKEND` changes make this conflict with `master`. Sorry about that! On a positive note, it should be pretty obvious how to resolve the conflicts :smile: